src/states/level_select.gd

```diff
@@ -63,15 +65,16 @@ func display() -> void:
 
 func _on_level_button_pressed(idx: int) -> void:
 	Global.current_level_idx = 9 * page_idx + idx
-	if Global.current_level_idx == 0:
-		Autosplitter.run_start()
-	var error := get_tree().change_scene_to_file("res://src/states/game.tscn")
-	assert(not error)
+	var game := preload("res://src/states/game.tscn").instantiate() as Game
+	get_tree().root.add_child(game)
+	get_tree().set_current_scene(game)
+	get_tree().root.remove_child(self.owner)
```

Comment:
I'm assuming this can't be done with `get_tree().change_scene_to_file(...)`?

Reply:
Yeah because we need to call `play_single_level` after the game scene is loaded in. Couldn't find a way to do that using `change_scene_to_file`